example.ipynb

```diff
@@ -92,66 +86,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
```

Comment:
If you can recompute the notebook easily to get the updated content, please do that. If that's too difficult at this point for some reason, then just leave it as is.